src/main/js/SaveApi.js

```diff
@@ -6,22 +6,23 @@ import { Fetch, getRestUrl, sseService, loadingIndicator, capabilityAugmenter }
 export class SaveApi {
 
     indexRepo(organization, teamName, repoName) {
-        const createUrl = `${getRestUrl({organization})}/pipelines/`;
-
+        const createUrl = repoName ? `${getRestUrl({organization})}pipelines/` : `${getRestUrl({organization, pipeline: teamName})}`;
```

Comment:
No, saving is happening at https://github.com/jenkinsci/blueocean-pipeline-editor-plugin/blob/master/src/main/js/EditorPage.jsx#L387. https://github.com/jenkinsci/blueocean-pipeline-editor-plugin/blob/master/src/main/js/EditorPage.jsx#L407 is round about way of triggering indexing for OrgFolder - that is GitHub/ghe case.